packages/components/react/src/Panels/PreviewPanel.tsx

```diff
@@ -61,6 +61,13 @@ export const PreviewPanel = memo(
       [],
     );
 
+    // we update the iframes position at ~30fps, needed for mobile
+    useEffect(() => {
+      const intervalId = setInterval(onResize, 20);
+
+      return () => clearInterval(intervalId);
+    }, []);
```

Comment:
With this code we make sure that the preview are correctly positioned. This is important on mobile when switching between the content and the editor.

I kept the existing `ResizeObserver` instances because they are smoother. 

This also fix the edge case where the preview stop moving when increasing the size of the terminal section.